tests/foreman/api/test_errata.py

```diff
@@ -0,0 +1,284 @@
+
+        @Setup: Errata synced on satellite server.
+
+        @Steps:
+
+        1. PUT /katello/api/systems/bulk/update_content
+
+        @Assert: errata is installed in the host-collection.
+
+        @Status: Manual
+
+        """
+
+    @unittest.skip(NOT_IMPLEMENTED)
+    def test_errata_list_1(self):
+        """@Test: View all errata specific to an Org
+
+        @Feature: Errata
+
+        @Setup: Errata synced on satellite server.
+
+        @Steps: Create two Orgs each having a product synced and contains
+        errata.
+
+        @Assert: Check that the errata belonging to one Org is not showing in
+        the other.
+
+        @Status: Manual
+
+        """
+
+    @unittest.skip(NOT_IMPLEMENTED)
+    def test_errata_list_2(self):
+        """@Test: View all errata in an Org sorted by Updated
+
+        @Feature: Errata
+
+        @Setup: Errata synced on satellite server.
+
+        @Steps:
+
+        1. GET /katello/api/errata
+
+        @Assert: Errata is filtered by Org and sorted by Updated date.
+
+        @Status: Manual
+
+        """
+
+    @unittest.skip(NOT_IMPLEMENTED)
+    def test_errata_list_3(self):
+        """@Test: Filter errata by CVE
+
+        @Feature: Errata
+
+        @Setup: Errata synced on satellite server.
+
+        @Steps:
+
+        1. GET /katello/api/errata
+
+        @Assert: Errata is filtered by CVE.
+
+        @Status: Manual
+
+        """
+
+    @unittest.skip(NOT_IMPLEMENTED)
+    def test_errata_systems_list_1(self):
+        """@Test: View a list of affected content hosts for an erratum
+
+        @Feature: Errata
+
+        @Setup: Errata synced on satellite server.
+
+        @Steps:
+
+        1. GET /katello/api/systems
+
+        @Assert: List of affected content hosts for the given erratum is
+        retrieved.
+
+        @Status: Manual
+
+        """
+
+    @unittest.skip(NOT_IMPLEMENTED)
+    def test_errata_systems_list_2(self):
+        """@Test: Filter errata list based on affected content hosts
+
+        @Feature: Errata
+
+        @Setup: Errata synced on satellite server.
+
+        @Steps:
+
+        1. GET /katello/api/errata
+
+        @Assert: Errata is filtered based on affected content hosts.
+
+        @Status: Manual
+
+        """
+
+    @unittest.skip(NOT_IMPLEMENTED)
+    def test_errata_sort_1(self):
+        """@Test: Filter errata by issued date
+
+        @Feature: Errata
+
+        @Setup: Errata synced on satellite server.
+
+        @Steps:
+
+        1. GET /katello/api/errata
+
+        @Assert: Errata is sorted by issued date.
+
+        @Status: Manual
+
+        """
+
+    @unittest.skip(NOT_IMPLEMENTED)
+    def test_errata_content_host_1(self):
+        """@Test: Filter applicable errata for a content host by current and
+        Library environments
+
+        @Feature: Errata
+
+        @Setup:
+
+        1. Make sure multiple environments are present.
+
+        2. One of Content host's previous environment has additional errata.
```

Comment:
This works fine, but the docstring can be made a little more compact by dropping the whitespace between list items. See example [here](http://rst.ninjs.org/?n=4bb0882de29394dbfedda0a372be75dd&theme=basic).